accounting/transferwise/tests/test_models.py

```diff
@@ -79,18 +79,6 @@ def test_create_payments(self, date):
         self.bulk_payment.create_payments()
         self.assertEqual(models.TransferWisePayment.objects.all().count(), 2)
 
-    def test_create_payments_which_already_exist(self):
-        """ Payments are not created because they already exist in the system. """
-        self._create_recipients(create_invoices=True)
-        for unused in range(3):
-            self.bulk_payment.create_payments()
-            self.assertEqual(models.TransferWisePayment.objects.all().count(), 2)
-
-    def test_create_payments_no_recipients(self):
-        """ Payments are not created because no valid recipients exist. """
-        self.bulk_payment.create_payments()
-        self.assertFalse(models.TransferWisePayment.objects.all().exists())
-
```

Comment:
@UmanShahzad Could you test the behavior of the changed code in https://github.com/open-craft/accounting/pull/9/commits/e37fd90bd31be5ba82fca709adc5adf1c61a798d#diff-107f6db16b31f879de8edfef974b4f2eR119  rather than just delete tests?

Reply:
The existing tests already do test the behavior, because factory-created invoices are unpaid. Do you mean to check if they got marked as paid successfully?

Reply:
@UmanShahzad What I meant is more that since no test were added or modified after `start_date` and `end_date` were added, so it means that their behavior isn't tested no? Ie that it only includes invoices from the specified period? 

Also, are all the invoices used in these tests unpaid? It seem important to test that paid invoices are not included, even if they are in the time period entered?

Reply:
@antoviaque Ah ok, that makes sense -- yeah I will test this behavior now.

Reply:
@antoviaque Covered in https://github.com/open-craft/accounting/pull/9/commits/817a7a372fb9959cdef4576fbf1445ddf971c53e